### PR TITLE
Newmachine

### PR DESCRIPTION
**Pull request by andreynav**
- added waitForTime method for debugging
- added new test case to tables spec file
- added initElement method
- fix for null error of textContent and cleaning code
- added method only for running only single test
- added new custom spec reporter

### CODE REVIEW

**Review by potapovDim — CHANGES_REQUESTED** [review]

File: machines/testing/lib/runner/runner.js [hunk 1]
```diff
@@ -1,42 +1,68 @@
...
 
 function wrappedIt(name, cb) {
 	global.it(name, async function () {
+		if (REPORTER == "SPEC") {
+			console.log(`\t`);
+		}
+		await wrapedTest(cb).call(this);
+	});
+}
+
```
Comment (potapovDim) [inline machines/testing/lib/runner/runner.js:23]: wrappedIt.only = wrappedOnly

**Review by potapovDim — CHANGES_REQUESTED** [review]

File: machines/testing/specs/index.spec.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ describe("Login sute", () => {
 	});
 
 	describe("Login to the app as exisitng user", async () => {
-		it("should login an exisiting user and redirect him to Tables page", async () => {
+		it.only("should login an exisiting user and redirect him to Tables page", async () => {
```
Comment (potapovDim) [inline machines/testing/specs/index.spec.js:17]: remove only

File: machines/testing/specs/table.spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	beforeEach("Executes beforeEach", async () => {
+		await browser.initBrowser("chromium", { headless: false, slowMo: 50 });
+	});
+
+	afterEach("Executes afterEach", async () => {
+		await browser.closeBrowser();
+	});
+
+	describe("Add new machine to table", () => {
+		it.only("should login as admin and add a new machine", async () => {
```
Comment (potapovDim) [inline machines/testing/specs/table.spec.js:17]: remove only

**Review by potapovDim — CHANGES_REQUESTED** [review]

File: machines/testing/specs/table.spec.js (hunk 3 above)
Comment (Andrei1982) [reply machines/testing/specs/table.spec.js]: removed

File: machines/testing/specs/index.spec.js (hunk 2 above)
Comment (Andrei1982) [reply machines/testing/specs/index.spec.js]: removed

**Review by potapovDim — APPROVED** [review]

**Pull request merged**